### DISCUSSION TITLE
FPS drops

### DISCUSSION THREAD

**Discussion by mshzsh** [hyprwm/Hyprland#1116] [Q&A] [answered]
After update Hyprland that I did just now, the FPS dropped drastically! for example, before the update, the FPS with the video background was 60 and after the update it reached 30.
Even without video background, when I open a float window on a window, the FPS drops below 30 for a few seconds.
Also when watching a movie, the FPS has a very sinusoidal performance.
I had last updated Hyprland about 10 days ago. (before now)

**Comment by vaxerski** [comment, marked as answer]
> about 10 days ago

oh god thats a while ago

can you try disabling blur new optimizations under decoration? you mentioned a video wallpaper, and those were made on by default recently. On animated wallpapers, they can increase lag.

**Reply by mshzsh** [reply]
The blur is disabled, I did not make any changes in the config file, and after the update, the FPS became 30, and then I disabled animated wallpaper then  FPS reached 60, but when using the programs it drops relatively sharply, which was not the case before

**Reply by mshzsh** [reply]
When open Guake terminal :

https://user-images.githubusercontent.com/41986782/204148108-2562b076-1e76-4cd8-97e2-9cae59bad99c.mp4

Watch the video in following post, when I open Guake but the FPS does not change. Despite the animated wallpaper! : 
[1114](https://github.com/hyprwm/Hyprland/issues/1114)
of course the most severe frame drop occurs when watching a movie!

**Reply by vaxerski** [reply]
do a git bisect then and provide the faulty commit

**Reply by mshzsh** [reply]
Do I have to restart Hyprland after testing each commit and build?

**Reply by JustSimplyKyle** [reply]
Yes, or alternatively use embedded hyprland sessions

**Reply by vaxerski** [reply]
I'd advise against nesting while testing performance issues.

**Reply by mshzsh** [reply]
Finally, the problem was solved by add `blur_new_optimizations=false` in config file.

You mentioned this point above, I thought that the` blur=false `option disables the entire blur

**Reply by vaxerski** [reply]
it... should. Anyways, it's weird that this happens...?

**Reply by vaxerski** [reply]
did you have an animated wallpaper for it to do this?

**Reply by mshzsh** [reply]
Yes, animated wallpaper made me notice the drop in performance.
Finally, after checkout to previous commits, I found [750eb76](https://github.com/hyprwm/Hyprland/commit/750eb76df305bb9aa77b6d7281eb9032ce4bf9fa) and realized that I should put `blur_new_optimizations=false` in the config file.

**Reply by vaxerski** [reply]
can you check with 3dd06b674a252c51937d20ffecfc0274ce63cfff

**Reply by mshzsh** [reply]
Without `blur_new_optimizations=false`?

**Reply by mshzsh** [reply]
Excellent! I commented `blur_new_optimizations=false` then updated Hyprland and everything was normal :+1:
